sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -95,7 +122,9 @@ export {
   AdministratorDetails,
```

Comment:
`AdministratorContact`

Reply:
Couldn't do this today. Please check the updated description of this PR.

Reply:
This has been handled in @jonathandturner's commit https://github.com/Azure/azure-sdk-for-js/pull/5991/commits/1ef22d22baf363462e791bd9064546c1208a0bdf